test/PostHogTest.php

```diff
@@ -59,6 +60,19 @@ public function testCapture(): void
         );
     }
 
+    public function testCaptureWithSendFeatureFlagsOption(): void
```

Comment:
what we're really testing here is that capture should include those feature flag properties if this option is true right?

Reply:
Adding a test to check the result of the capture would add some boilerplate so I just did a simple check to see if the decide endpoint was hit in this code path which I think should be good enough.